dev/sensor/htu21d/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_library(htu21d INTERFACE)
```

Comment:
Specifically, you must specify that this library depends on `types` library which contains `enum class err`:
`target_link_libraries(htu21d INTERFACE types)